docs/documentation/server_admin/topics/organizations/authenticating-members.adoc

```diff
@@ -0,0 +1,99 @@
+[id="authenticating-members_{context}"]
+
+=  Authenticating members
+[role="_abstract"]
+One of the main changes introduced by organizations when enabled to a realm is on how users authenticate.
+These changes are mainly about being able to recognize if a user is authenticating in the context of an organization
+and changing the authentication flow accordingly on a per-organization basis.
+
+When a realm is created, the authentication flows are automatically updated to enable specific steps to authenticate and onboard organization members. The authentication flows updated are:
+
+* `browser`
+
+* `first broker login`
+
+The main change to the `browser` flow is that it defaults to an identity-first login so that users are identified before prompting for their credentials.
+In regard to the first broker login flow, the main change there is to automatically add the user as an organization member once they authenticate through the identity provider associated with an organization and successfully complete flow.
+
+The decision to whether or not an identity-first login should happen is based on the availability of any organization in a realm.
+If no organizations exist yet, the user will follow the usual steps to authenticate using both username and password,
+or any other step configured in the browser flow. Otherwise, the user will be asked first for a username or email to continue authenticating to a realm.
```

Comment:
```suggestion
The choice to use an identity-first login or not depends on the existence of an organization in a realm.
If no organizations exist yet, the user follows the usual steps to authenticate using the username and password,
or any other step configured in the browser flow. Otherwise, the user is asked first for a username or email to continue authenticating to a realm.
```